### PR TITLE
 Adding Random sampling to Numpy Frontend: logseries 

### LINKED ISSUE #21402: logseries

### PR DESCRIPTION
**Pull request by Tanzeel161**
# PR Description 

## Related Issue 

Close #21402 

## Checklist 

- [x] Did you add a function?
- [x] Did you add the tests?
- [x] Did you follow the steps we provided?

### CODE REVIEW

**Comment by KareemMAX** [comment]
ivy-gardener

**Review by KareemMAX — CHANGES_REQUESTED** [review]
Hey @Tanzeel161,
I've added some comments that I would like you to address. After that we can run the final tests

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_random/test_functions.py [hunk 1]
```diff
@@ -1019,3 +1019,41 @@ def test_numpy_weibull(
...
+
+@handle_frontend_test(
+    fn_tree="numpy.random.logseries",
+    input_dtypes=helpers.get_dtypes("float", index=2),
+    p=st.floats(
+        allow_nan=False,
+        allow_infinity=False,
+        width=32,
+        min_value=0,
+        exclude_min=True,
```
[the remark this answers is not in the document]
Comment (Tanzeel161) [reply ivy_tests/test_ivy/test_frontends/test_numpy/test_random/test_functions.py]: Hi @KareemMAX , i have made the requested changes. Please approve the workflows. Thanks for the help!!

**Review by KareemMAX — APPROVED** [review]
LGTM, merging now 😄

**Pull request merged**